sycl/test-e2e/Plugin/interop-level-zero-image.cpp

```diff
@@ -3,7 +3,8 @@
 // RUN: %{run} %t.out
 
 // spir-v gen for legacy images at O0 not working
-// UNSUPPORTED: O0
+// UNSUPPORTED: *
```

Comment:
I don't think you can do *, we see errors [here](https://github.com/intel/llvm/actions/runs/8378865540/job/22945561183?pr=13091). 

If it only fails on arc we could do

`// UNSUPPORTED: gpu-intel-dg2'

otherwise maybe

`// UNSUPPORTED: gpu`

Reply:
You're right. `UNSUPPORTED: *` only works when the test is `XFAIL`. I'll try the gpu-intel-dg2